examples/flutter_gallery/lib/demo/material/cards_demo.dart

```diff
@@ -59,102 +79,260 @@ class TravelDestinationItem extends StatelessWidget {
+                  splashColor: Theme.of(context).colorScheme.primary.withAlpha(30),
+                  child: Stack(
+                    children: <Widget>[
+                      TravelDestinationContent(destination: widget.destination),
+                      Opacity(
+                        opacity: _isSelected ? 1 : 0,
+                        child: Container(
+                          color: Theme.of(context).colorScheme.primary.withAlpha(41),
+                        ),
                       ),
-                    ),
-                    Positioned(
-                      bottom: 16.0,
-                      left: 16.0,
-                      right: 16.0,
-                      child: FittedBox(
-                        fit: BoxFit.scaleDown,
-                        alignment: Alignment.centerLeft,
-                        child: Text(destination.title,
-                          style: titleStyle,
+                      Opacity(
+                        opacity: _isSelected ? 1 : 0,
+                        child: const Align(
+                            alignment: Alignment.topRight,
+                            child: Padding(
+                              padding: EdgeInsets.all(4.0),
+                              child: Icon(Icons.check_circle, color: Colors.white,),
+                            )
                         ),
                       ),
-                    ),
-                  ],
+                    ],
+                  )
                 ),
               ),
-              // description and share/explore buttons
-              Expanded(
-                child: Padding(
-                  padding: const EdgeInsets.fromLTRB(16.0, 16.0, 16.0, 0.0),
-                  child: DefaultTextStyle(
-                    softWrap: false,
-                    overflow: TextOverflow.ellipsis,
-                    style: descriptionStyle,
-                    child: Column(
-                      crossAxisAlignment: CrossAxisAlignment.start,
-                      children: <Widget>[
-                        // three line description
-                        Padding(
-                          padding: const EdgeInsets.only(bottom: 8.0),
-                          child: Text(
-                            destination.description[0],
-                            style: descriptionStyle.copyWith(color: Colors.black54),
-                          ),
-                        ),
-                        Text(destination.description[1]),
-                        Text(destination.description[2]),
-                      ],
-                    ),
-                  ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class SectionTitle extends StatelessWidget {
+  const SectionTitle({
+    Key key,
+    this.title
+  }) : super(key: key);
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(4, 4, 4, 12),
```

Comment:
Nit: As far as I know, Flutter library pattern is to keep the `.0` on doubles, so:

`const EdgeInsets.fromLTRB(4.0, 4.0, 4.0, 12.0),`

Reply:
done